include/crm/common/iso8601_internal.h

```diff
@@ -25,6 +25,7 @@ pcmk__time_hr_t *pcmk__time_hr_new(const char *date_time);
 void pcmk__time_hr_free(pcmk__time_hr_t *hr_dt);
 char *pcmk__time_format_hr(const char *format, const pcmk__time_hr_t *hr_dt);
 char *pcmk__epoch2str(const time_t *source, uint32_t flags);
+char *pcmk__timespec2str(const struct timespec *ts, uint32_t flags);
```

Comment:
I had created a `pcmk__time_hr_as_string()` function and a `pcmk__time_hr_set_timespec()` function, but by the time I was done with this they no longer had any callers, so I removed them. At this point, they're not too complicated to add back (about 10 lines each) if we ever want them.